src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java

```diff
@@ -480,6 +481,21 @@ public void canRunLLVM(Triplet triplet) throws IOException, InterruptedException
                         "$GRAALVM_HOME/bin/gu install llvm-toolchain");
             }
         }
+
+        // TODO: Remove when GraalVM has support for Xcode 12
+        if (new Triplet(Constants.Profile.IOS).equals(triplet)) {
+            Path gluonversion = graalPath.resolve("gluonversion.txt");
+            if (!Files.exists(gluonversion) ||
+                    !Files.readAllLines(gluonversion).contains("graalvm-ce-java11-20.2.0.hotfix-xcode12")) {
+                XcodeUtils xcodeUtils = new XcodeUtils(XcodeUtils.SDKS.IPHONEOS);
+                if (xcodeUtils.getDTXcode().startsWith("12")) {
+                    throw new IOException("\n\n\nYour current GraalVM build can't build for iOS with Xcode 12+.\n\n" +
```

Comment:
`11+` also includes version 12. I would remove the `+` suffixes in the message.

Reply:
Where does `12+` relate to 11? I'll remove it anyway

Reply:
I was in fact referring to the 11+ in the next line. So having 12+ indeed makes a bit of sense, but let us hope we can remove this code before xcode 13.

Reply:
Oh, my bad, fixed it.